packages/client/src/forms/register/mappings/mutation/field-mappings.ts

```diff
@@ -177,6 +177,7 @@ export const addressMutationTransformer =
     useCase?: string
     lineNumber?: number
     transformedFieldName?: string
+    isLowestAdministrativeLevel?: boolean
```

Comment:
Do countries need to add this option explicitly to their country config once this is merged?

Reply:
Yeah. I'll add a note to this PR description and there should be a comment in the Farajaland PR.